src/Symfony/Component/Notifier/Bridge/SmsFactor/composer.json

```
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/smsfactor-notifier",
```

```suggestion
    "name": "symfony/smsfactor-notifier",
```
@fabpot are you fine with the package name?

---

Must be `sms-factor-notifier` as the class name is `SmsFactor` and not `Smsfactor`